<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BlockBrowser</class>
 <widget class="QDialog" name="BlockBrowser">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>539</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>GalaxyCash Block Explorer</string>
  </property>
  <layout class="QGridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetMaximumSize</enum>
   </property>
   <property name="leftMargin">
    <number>21</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <property name="autoFillBackground">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="layoutWidgetBE1">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>822</width>
        <height>27</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayoutBE">
       <property name="spacing">
        <number>7</number>
       </property>
       <item>
        <widget class="QLabel" name="labelBlockExplorer">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true">font-weight:bold;</string>
         </property>
         <property name="text">
          <string>Block Explorer</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacerBE">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidgetBE2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>30</y>
        <width>821</width>
        <height>481</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayoutBE" rowstretch="0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0" columnstretch="0,0,0">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <property name="horizontalSpacing">
        <number>7</number>
       </property>
       <item row="13" column="2">
        <widget class="QLabel" name="inputsBox">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="15" column="2">
        <widget class="QLabel" name="outputsBox">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="lineSearch"/>
       </item>
       <item row="1" column="2">
        <widget class="QPushButton" name="searchButton">
         <property name="text">
          <string>Search</string>
         </property>
        </widget>
       </item>
       <item row="13" column="1">
        <widget class="QLabel" name="inputsLabel">
         <property name="text">
          <string>Inputs:</string>
         </property>
        </widget>
       </item>
       <item row="20" column="1" colspan="2">
        <widget class="QPushButton" name="backButton">
         <property name="text">
          <string>Back</string>
         </property>
        </widget>
       </item>
       <item row="15" column="1">
        <widget class="QLabel" name="outputsLabel">
         <property name="text">
          <string>Outputs:</string>
         </property>
        </widget>
       </item>
       <item row="21" column="1" colspan="2">
        <widget class="QPushButton" name="closeButton">
         <property name="text">
          <string>Close</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QLabel" name="heightLabelBE1">
         <property name="text">
          <string>0</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="7" column="2">
        <widget class="QLabel" name="merkleBox">
         <property name="minimumSize">
          <size>
           <width>187</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0x0</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="9" column="1">
        <widget class="QLabel" name="bitsLabel">
         <property name="text">
          <string>Bits:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="hashLabel">
         <property name="text">
          <string>Hash:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="1">
        <widget class="QLabel" name="diffLabel">
         <property name="text">
          <string>Difficulty:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLabel" name="hashBox">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="text">
          <string>0x0</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="10" column="1">
        <widget class="QLabel" name="timeLabel">
         <property name="text">
          <string>Timestamp:</string>
         </property>
        </widget>
       </item>
       <item row="19" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="12" column="2">
        <widget class="QLabel" name="moneySupplyBox">
         <property name="text">
          <string>0.00 GCH</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="12" column="1">
        <widget class="QLabel" name="moneySupplyLabel">
         <property name="text">
          <string>Money Supply:</string>
         </property>
        </widget>
       </item>
       <item row="11" column="2">
        <widget class="QLabel" name="diffBox">
         <property name="text">
          <string>0.00</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="merkleLabel">
         <property name="text">
          <string>Merkle:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="2">
        <widget class="QLabel" name="timeBox">
         <property name="text">
          <string>0</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="heightLabelBE2">
         <property name="text">
          <string>Height:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1">
        <widget class="QLabel" name="nonceLabel">
         <property name="text">
          <string>Nonce:</string>
         </property>
        </widget>
       </item>
       <item row="8" column="2">
        <widget class="QLabel" name="nonceBox">
         <property name="text">
          <string>0</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="typeLabel">
         <property name="text">
          <string>Type:</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2">
        <widget class="QLabel" name="typeBox">
         <property name="text">
          <string>PoW</string>
         </property>
        </widget>
       </item>
       <item row="18" column="1">
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="2">
        <widget class="QLabel" name="bitsBox">
         <property name="text">
          <string>0</string>
         </property>
         <property name="textInteractionFlags">
          <set>Qt::LinksAccessibleByMouse|Qt::TextSelectableByMouse</set>
         </property>
        </widget>
       </item>
       <item row="6" column="2">
        <widget class="QLabel" name="algoBox">
         <property name="text">
          <string>x12</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="algoLabel">
         <property name="text">
          <string>Algorithm:</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
